### ISSUE TITLE
[ADE Support] Collaborate and share knowledge with MHV Accessibility specialists - set up recurring meeting

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#131623]
## Description of the Work
ADE met with MHV a11y specialists to share knowledge, best practices, discuss opportunities to collaborate and identify next steps. As a result:
- MHV team was able to generate a list of services similar to ADE list of services
- ADE and MHV agreed to refer all request for research support to ADE (via the MHV team/ensuring the MHV team is aware of the request being made) 
- MHV will reach out for additional support when required
- Private slack channel set up to discuss a11y issues when required

## ADE Tasks
- [x] Collaborate `task`
- [ ] ADE Support `task`
- [ ] Document `task`
- [ ] Review `task`
